docs/serving/samples/autoscale-go/README.md

```diff
@@ -118,7 +118,7 @@ A demonstration of the autoscaling capabilities of a Knative Serving Revision.
 
 Knative Serving autoscaling is based on the average number of in-flight requests
 per pod (concurrency). The system has a default
-[target concurrency of 100](https://github.com/knative/serving/blob/master/config/core/configmaps/autoscaler.yaml#L54-L60)
+[target concurrency of 70](https://github.com/knative/serving/blob/8c7134ebfe368ee178229e99166e5bff8167b8ed/config/core/configmaps/autoscaler.yaml#L40-L55)
```

Comment:
We should probably say the the system has a default target concurrency of 100 and a default target utilization of 70%, effectively making it 70 from the autoscaler's PoV.

Reply:
So change the 70 back to 100? I think I just misread it. Looks like @vagababov said the same thing.

Reply:
Yes, the proper way to phrase is, I think:
keep this hundred and put link to the proper variable (https://github.com/knative/serving/blob/8c7134ebfe368ee178229e99166e5bff8167b8ed/config/core/configmaps/autoscaler.yaml#L74) and then mention the variable your link points to with default of 70% bringing us to the _default effective target_ of 70.

Reply:
I reverted back to 100 and updated the link, but am confused by what we need to get across in this example. Talking about the 70 percentage might confuse people, it confused me because I thought that was the actual target being used instead of default.